.github/workflows/flutter-ci.yml

```
@@ -21,7 +21,7 @@ jobs:
           flutter-version: '2.8.x'
           channel: "stable"
       - name: Create .env file
-        run: echo "DEVELOPMENTMODE=true" > .env
+        run: echo "DEVELOPMENTMODE=true" and "NEWSURL= https://www.freecodecamp.org/news/ghost/api/v3/content/" > .env
```

This will not work as the `env` fill have the text `DEVELOPMENTMODE=true and NEWSURL= https://www.freecodecamp.org/news/ghost/api/v3/content/` which isn't the right format

Instead echo only one env var at a time like this:
```suggestion
        run: |
          echo "DEVELOPMENTMODE=true" > .env
          echo "NEWSURL=https://www.freecodecamp.org/news/ghost/api/v3/content/" >> .env
```

Note that only the **first** echo command should use `>` operator while the following commands should use `>>` operator